Gemfile

```diff
@@ -1,7 +1,4 @@
-source 'https://rubygems.org'
-#this will probably not be available on rubygems, at least initially
-#will probably be only available at https://rubygems.delivery.puppetlabs.net
-#TODO: Change this line when it is actually available internally
+source 'http://rubygems.delivery.puppetlabs.net'
```

Comment:
i don't really understand the above comments.  this is the source for this project's/Gem's _dependencies_, not where it lives.  i see it's dependencies as mostly just beaker, which is available on rubygems, along with the dev. deps